jupyter_sphinx/execute.py

```diff
@@ -384,6 +439,73 @@ def execute_cells(kernel_name, cells, execute_kwargs):
     return notebook
 
 
+def get_widgets(notebook):
+    try:
+        return notebook.metadata.widgets[WIDGET_STATE_MIMETYPE]
+    except AttributeError:
+        # Don't catch KeyError, as it's a bug if 'widgets' does
+        # not contain 'WIDGET_STATE_MIMETYPE'
+        return None
+
+
+def contains_widgets(notebook):
+    widgets = get_widgets(notebook)
+    return widgets and widgets['state']
+
+
+# TODO: Remove this once  https://github.com/jupyter/nbconvert/pull/900
```

Comment:
This has now happened; we should rely on the default behavior of the `ExecutePreprocessor` now.